test/index.test.js

```diff
@@ -577,6 +577,22 @@ describe('Elasticsearch Service', () => {
             expect(results.test.mappings.mobiles._parent.type).to.equal('people');
           });
       });
+
+      it('should return a promise when the passed in method is not defined', () => {
+        expect(app
+          .service('mobiles')
+          .raw(undefined, {})
+          .catch(err => 'Error is:' + err)
```

Comment:
I think `.catch` is enough to prove it's a promise and `.toString` below is not necessary.